ios/FluentUI/Command Bar/CommandBar.swift

```diff
@@ -38,12 +39,18 @@ open class CommandBar: UIView {
             self.trailingButton = button(forItem: trailingItem, isPersistSelection: false)
         }
 
+        backgroundColor = commandBarTokens.backgroundColor
         translatesAutoresizingMaskIntoConstraints = false
 
         configureHierarchy()
         updateButtonsState()
     }
 
+    open override func didMoveToWindow() {
+        super.didMoveToWindow()
+        commandBarTokens.didChangeAppearanceProxy()
```

Comment:
Hey @sophialee0416. While reviewing the tokenization of the NotificationView earlier today I made this comment that I missed in the CommandBar review:

Could you create a quick PR to update this call to updateForCurrentTheme() instead?
Thanks!

https://github.com/microsoft/fluentui-apple/pull/715#discussion_r718039047